lib/archethic/networking/ip_lookup/nat_discovery/miniupnp.ex

```diff
@@ -3,11 +3,13 @@ defmodule Archethic.Networking.IPLookup.NATDiscovery.MiniUPNP do
 
   require Logger
 
-  @upnpc Application.app_dir(:archethic, "priv/c_dist/upnpc")
+  def upnpc() do
```

Comment:
Why do you need to make it as function ?

Reply:
because the path change between compile time and run time as we cp the resulting tar gz from ci container to cd containers.

Reply:
Is the UPnP still called/needed on the docker releases ?

Reply:
normally no, but I did this for other binaries as well, should I make revert the change on upnpc while keeping the change on the others ?

Reply:
no harm, maybe make the function private